Privacy.md

```diff
@@ -0,0 +1,85 @@
+
+- Ability to have full network access which means the app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- Control vibration: This app can control your device vibrator.
+
+The above permissions can be revoked at any time you desire, but revoking these permissions will cause the software to cease to function. These permissions work hand-in-hand simultaneously to ensure smooth operations and utilization of the software. This is privy to only users with devices running Android 6.0 upwards. <hr>
+
+## Crash reports sending
+
+Fruit Radar is stable and works fine but you may still experience few errors(bugs) and crashes, information about these crashes are important to me(the developer) so the issues can be fixed immediately and the software is running efficiently again. If you encounter a bug during the use of this software, please open an [issue](https://github.com/niccokunzmann/mundraub-android/issues/new) on GitHub and it will be fixed in due time. You can also access the project's repository directly from with the app to view and open new issues.
+You can report a debug log file if you gave EXTERNAL_STORAGE permissions or use logcat to retrieve the log. If you gave permissions, you should be able to find it under eu.quelltext.mundraub.log.txt or eu.quelltext.mundraub.error.txt. The app (if starting) should point you to the file.
+You can use [MatLog Libre](https://f-droid.org/en/packages/com.pluscubed.matloglibre/) to retrieve the app log before and after a crash. <hr>
+
+## Service Providers
+
+Fruit Radar is dependent upon the services of third-party services to facilitate its operations and to perform service-related services or to assist in analyzing how the services of the software are used. These third parties have access to your Personal Data only to perform these tasks on the developer's behalf and are obligated not to disclose or use it for any other purpose.
+
+The third-party tools/services that are employed by the developer and embedded in the software are;
+
+1. [Mundraub.org](https://mundraub.org/) (German)
+2. [Fruitmap.org](https://www.fruitmap.org/) (English, Slovakian, Czech) and 
+3. [Na-Ovoce.cz](https://na-ovoce.cz/)  (Czech)
+
+These websites are third party websites which house the plant database or plant sources. From these sources, plants details are downloaded.
+
+[Mundraub.org](https://mundraub.org/) collects your personal information only when you submit your data during registration. Read the company's [Privacy policy](https://mundraub.org/agb_datenschutz) 
+
+[Mundraub.org](https://mundraub.org/) and [Na-Ovoce.cz](https://na-ovoce.cz/) reveals text, position, images, email, username, category and password. They are also uploading platforms, so when you tell the app to upload/login/register, more data is uploaded to this platforms. The password is saved in the app.
```

Comment:
- reveals -> reveal
- password is not revealed
- password is saved in the app and on the platform.

Reply:
I think reveals is the correct word to use in that context.